setting-rate-limit-cloud-api.html.md.erb

```diff
@@ -38,9 +38,43 @@ Use the following properties in the `cloud_controller_ng` BOSH job to define rat
 	<td>The interval in minutes until a user's available CAPI requests reset</td>
     <td>60</td>
  </tr>
+ <tr>
+	<td>cc.rate_limiter_v2_api.enabled</td>
+	<td>Enable rate limiting for UAA-authenticated V2 API (v2/*, except v2/info) endpoints per user or client</td>
+    <td>false</td>
+ </tr>
+ <tr>
+	<td>cc.rate_limiter_v2_api.general_limit</td>
+	<td>The number of requests a user or client is allowed to make for v2/* endpoints that do not have a custom limit over the configured interval</td>
+    <td>10000</td>
```

Comment:
The defaults don't match the [configuration in capi-release](https://github.com/cloudfoundry/capi-release/blob/develop/jobs/cloud_controller_ng/spec#L1044-L1055):
- cc.rate_limiter_v2_api.general_limit: 2000
- cc.rate_limiter_v2_api.admin_limit: 2000